src/Voron/Impl/Backup/StreamExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.IO;
+using System.Threading;
+
+namespace Voron.Impl.Backup
+{
+    public static class StreamExtensions
+    {
+        private const int DefaultBufferSize = 81920;
+
+        public static void CopyTo(this Stream source, Stream destination, CancellationToken cancellationToken)
+        {
+            var buffer = new byte[DefaultBufferSize];
```

Comment:
Move this to be a thread static value, no need to keep creating it on every call.